laterpay/application/Form/EnabledPostType.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * LaterPay enabled post types form class.
+ *
+ * Plugin Name: LaterPay
+ * Plugin URI: https://github.com/laterpay/laterpay-wordpress-plugin
+ * Author URI: https://laterpay.net/
+ */
+class LaterPay_Form_EnabledPostType extends LaterPay_Form_Abstract
+{
+
+    /**
+     * Implementation of abstract method.
+     *
+     * @return void
+     */
+    public function init() {
+
+        $this->set_field(
+            'form',
+            array(
+                'validators' => array(
+                    'is_string',
+                    'cmp' => array(
+                        array(
+                            'like' => 'update_enabled_post_types',
```

Comment:
Does it compare it with existing post type? 
If not can we compare with all current post type just to be safe? `get_post_types( array( ), 'objects' );`